src/card/stories.js

```diff
@@ -0,0 +1,130 @@
+/* @flow */
+/*global module */
+import React from 'react';
+
+import {storiesOf} from '@storybook/react';
+
+import {LightTheme} from '../themes';
+import {ThemeProvider} from '../styles';
+
+import BaseCard from './card';
+import {
+  Action as CardAction,
+  Body as CardBody,
+  Thumbnail as CardThumbnail,
+  Title as CardTitle,
+} from './styled-components';
+
+import {thumbnail as thumbnailImg, header as headerImg} from './images';
+
+import type {Props} from './card';
+
+const cardWidth = '328px';
+const placeholderTitle = 'Card Title Entry';
+const placeholderText =
+  'Proin ut dui sed metus pharetra hend rerit vel non mi. Nulla ornare faucibus ex, non facilisis nisl. Maecenas aliquet mauris ut tempus cursus. Etiam semper luctus sem ac blandit. ';
+
+const Card = (props: Props) => (
+  <BaseCard style={{width: cardWidth}} {...props} />
+);
+Card.defaultProps = {
+  hasThumbnail: (props: {+thumbnail?: string}) => Boolean(props.thumbnail),
+  overrides: {},
+};
+
+function Button(props) {
+  return <button {...props} />;
+}
+
+storiesOf('Card', module)
+  .addDecorator(story => (
```

Comment:
can you add the RFC as the readme? just like here: https://github.com/uber-web/baseui/blob/master/src/select/stories.js#L19